docs/migration_guide.rst

```diff
@@ -1,6 +1,18 @@
 Migration guide
 ===============
 
+**********************
+0.20.0 Migration guide
+**********************
+
+This update adds support for python 3.12
+
+Changes in the :class:`~starknet_py.net.account.account.Account`:
+
+.. currentmodule:: starknet_py.net.account.account
+
+- :meth:`~Account.estimate_fee` has been added
```

Comment:
I would recommend against adding this information here, as it doesn't align to the context of the migration guide. Instead, it could be placed elsewhere:
- In the release notes
- In the guide under [Transaction Fee](https://starknetpy.readthedocs.io/en/latest/guide/account_and_client.html#transaction-fee) or as a separate section

Reply:
I added this in https://github.com/software-mansion/starknet.py/pull/1290/files